### PR TITLE
Widgets: Change imports to AnyQt

### PR DESCRIPTION
**Pull request by rokgomiscek**
Widgets: Change imports to AnyQt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-numpy==1.8.0
-numpydoc==0.4
-networkx==1.8.1
+numpy>=1.8.0
+numpydoc>=0.4
+networkx>=1.8.1
```
Comment (kernc) [inline requirements.txt]: My god! For a second there I thought this is why the add-on was uninstallable in certain situations ...

This file isn't used. Please remove it.

Comment (ajdapretnar) [inline requirements.txt]: Should we not have a requirements.txt as all other add-ons? I know it's not used, but I see it as a nice statement to what the add-on needs for running properly. Can we include it somehow or should we indeed obliterate it?

Comment (kernc) [inline requirements.txt]: It is a nice statement (although @astaric might disagree). When it's not this file, it's _setup.py_. Whatever way is fine by me, but keeping this file while it's being unused is woefully misleading.

Comment (astaric) [inline requirements.txt]: +1 for only keeping the requirements in the setup.py.

See INSTALL_REQUIRES and EXTRAS_REQUIRE. While at it, this might be the time to change PyQt4 requirement in setup.py to AnyQt

File: orangecontrib/network/widgets/graphview.py [hunk 2]
```diff
@@ -333,8 +335,8 @@ def addEdge(self, edge):
         self.scene().addItem(edge)
 
     def wheelEvent(self, event):
-        if event.orientation() != Qt.Vertical: return
-        self.scaleView(2**(event.delta() / 240))
+        #if event.orientation() != Qt.Vertical: return
```
Comment (astaric) [inline orangecontrib/network/widgets/graphview.py]: Was  this supposed to be removed?

File: orangecontrib/network/widgets/OWNxFile.py [hunk 3]
```diff
@@ -81,7 +80,7 @@ def reload_data(self):
     def populate_comboboxes(self):
         self.filecombo.clear()
         for file in self.recentFiles or (NONE,):
-            self.filecombo.addItem(path.basename(file))
+            self.filecombo.addItem(path.basename(file[0]))
```
Comment (astaric) [inline orangecontrib/network/widgets/OWNxFile.py]: Could you elaborate the addition of [0] here and in couple of changes below? If it is unrelated to the move to AnyQt, it might deserve a separate commit describing why it was changed.

Comment (ales-erjavec) [inline orangecontrib/network/widgets/OWNxFile.py]: Probably due to the changed (PyQt5 compatible) signature of QFileDialog.getOpenFile. If so it is best to change the code at the QFileDialog.getOpenFile [call site](https://github.com/rokgomiscek/orange3-network/blob/a8bebf9d618d2421165b08e77cd4b688e4a5e0d6/orangecontrib/network/widgets/OWNxFile.py#L217) (i.e. `filename, _ = QFileDialog.getOpenFile(...)`. In fact this is necessary judging from the code following it.

File: requirements.txt (hunk 1 above)
Comment (rokgomiscek) [inline requirements.txt]: Removed requirements.txt and added the requirements to setup.py

File: orangecontrib/network/widgets/graphview.py (hunk 2 above)
Comment (rokgomiscek) [inline orangecontrib/network/widgets/graphview.py]: No, changed so it works with PyQt5.

File: orangecontrib/network/widgets/OWNxFile.py (hunk 3 above)
Comment (rokgomiscek) [inline orangecontrib/network/widgets/OWNxFile.py]: Thank you. Fixed.

**Comment by ajdapretnar** [comment]
Looks ok to me. @rokgomiscek Could you perhaps just squash two commits into one if possible? That's generally desirable for a better commit history! Thanks!!!

**Comment by ajdapretnar** [comment]
Oh, I guess now is worse.... 😞 My previous commit got into this one. I think this is the case where you need to rebase.

**Comment by rokgomiscek** [comment]
Should be ok now.
